includes/post-list-functions.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Custom post list layouts
+ */
+
+if ( ! function_exists( 'research_clusters_layout_content' ) ) {
+	/**
+	 * Displays a list of faculty clusters
+	 * @author Jim Barnes
+	 * @since 1.0.0
+	 * @param string $retval The default markup
+	 * @param array $posts The array of posts
+	 * @param array $atts The shortcode attributes
+	 * @return string The modified markup
+	 */
+	function research_clusters_layout_content( $retval, $posts, $atts ) {
+		ob_start();
+
+		foreach( $posts as $idx => $post ) :
+
+		$bg_image_sm    = research_cluster_get_header_image( $post->ID );
+		$bg_image_xs    = research_cluster_get_header_image( $post->ID, true );
+
+		$short_desc  = get_field( 'cluster_short_desc', $post->ID ) ?: null;
+
+		$push = $idx % 2 === 0 ? false : true;
+
+	?>
+		<section id="<?php echo $post->post_name; ?>" aria-labelledby="<?php echo $post->post_name ?>-heading">
+			<div class="jumbotron jumbotron-fluid co-jumbotron-wrap media-background-container bg-inverse d-flex flex-column justify-content-end justify-content-md-center mb-0 p-0 p-sm-5">
+				<?php if ( $bg_image_sm && $bg_image_xs ) : ?>
+				<picture>
+					<source media="(min-width: 575px)" srcset="<?php echo $bg_image_sm; ?>">
+					<img src="<?php echo $bg_image_xs; ?>" alt="" class="co-jumbotron-bg media-background object-fit-cover">
+				</picture>
+				<?php endif; ?>
+				<div class="container">
+					<div class="row p-lg-5">
+						<div class="col-12 col-md-8<?php echo $push ? ' offset-md-4' : ''?>">
+							<div class="bg-inverse-t-3 p-5">
+								<h2 class="display-5 font-condensed text-uppercase mt-0 mb-4" id="<?php echo $post->post_name; ?>-heading" aria-label="<?php echo $post->post_title; ?>"><?php echo $post->post_title; ?></h2>
```

Comment:
`display-5` class doesn't exist

Reply:
Removed the class for now. We can use another sizing class if we need to.